modules/covid_vaccine/app/jobs/covid_vaccine/expanded_scheduled_submission_job.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+
+module CovidVaccine
+  class ExpandedScheduledSubmissionJob
+    include Sidekiq::Worker
+    include SentryLogging
+    sidekiq_options retry: false
+
+    def perform
+      Rails.logger.info('Covid_Vaccine Expanded_Scheduled_Submission: Start')
+
+      count = CovidVaccine::V0::ExpandedRegistrationSubmission.where(state: 'enrollment_pending').count
+      Rails.logger.info('Covid_Vaccine Expanded_Scheduled_Submission Start: enrollment_pending count', count: count)
+
+    # Sorting by DESC here because currently if a record fails MPI lookup, it remains in state=enrollment_pending,
+    # so starting at the beginning we may never see the new records as only the MPI error records would process due to limit
+    # with anticipated volume and execution every 15 minutes we should be able to process all new records 
+    # todo: implement task to find and deal with MPI error records and enrollment_out_of_bounds records
+      CovidVaccine::V0::ExpandedRegistrationSubmission.where(state: 'enrollment_pending').order('created_at DESC').limit(1000).map do |submission|
+        CovidVaccine::ExpandedSubmissionJob.perform_async(submission.id)
+      end
+
+      count = CovidVaccine::V0::ExpandedRegistrationSubmission.where(state: 'enrollment_pending').count
```

Comment:
Does getting the count here depend on the `CovidVaccine::ExpandedSubmissionJob`s above being done? Due to their async-ness that's not guaranteed.

Reply:
@kreek yep it sure does.  great point and that count will not work.  maybe we can have a separate scheduled job that runs a couple of times a day to report status of all states in DB (can reuse the [state_summary ](https://github.com/department-of-veterans-affairs/vets-api/blob/master/rakelib/covid_vaccine.rake#L4)rake task code)

Reply:
+1 I think if we want regular counts of status showing up in logs, that is a separate process. Maybe the only logging here beyond exceptions is the number of records iterated over on ln 21-23.

Reply:
@batemapf have not implemented logging to indicate number of records iterated over yet. Can look into that with a separate smaller PR